radiolog.py

```diff
@@ -302,6 +302,25 @@
 # #############################################################################
 # #############################################################################
 
+from PyQt5.QtCore import QSettings, QStandardPaths
```

Comment:
All the ui imports do need to go below the ui and qrc rebuild code, currently line 569 of master.  That was to accommodate #471